config/environments/production.rb

```diff
@@ -1,13 +1,13 @@
 Rails.application.configure do
-  # Settings specified here will take precedence over those in config/application.rb.
 
-  # Code is not reloaded between requests.
+
+  config.serve_static_assets = true
+  config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.